modules/arxiv.py

```diff
@@ -13,22 +12,23 @@
 
 import web, re, feedparser
 from web import urllib
-
-metamark_api = r'http://metamark.net/api/rest/simple'
+from modules.url import short
 
 # Base api query url
 base_url = 'http://export.arxiv.org/api/query?';
 
-request = 'search_query=all:{}&start=0&max_results=1'
+request = 'search_query={}&start=0&max_results=1'
 
 feedparser._FeedParserMixin.namespaces['http://a9.com/-/spec/opensearch/1.1/'] = 'opensearch'
 feedparser._FeedParserMixin.namespaces['http://arxiv.org/schemas/atom'] = 'arxiv'
 
-id_filter = re.compile(r'.*\/abs\/((?:[\d\.]{9})|(?:[\w\-\.]*\/\d{7}))(?:v\d+){,1}[/]{,1}') # id will be in \1
+id_filter = re.compile(r'.*\/abs\/((?:[\d\.]{9})|(?:[\w\-\.]*\/\d{7}))(?:v\d+){,1}[/]{,1}') # id will be in \1 
```

Comment:
You should add a rule to trim whitespace to your IDE so you don't include items like this in diffs.